cmake/thirdparty/FindCUDA.cmake

```diff
@@ -1872,7 +1860,7 @@ macro(CUDA_ADD_EXECUTABLE cuda_target)
       # We need to link with nvcc. Host linking prevents calling device methods in one library from kernels in another (yes, even with separable
       # compilation)
       string (REPLACE ";" " " LD_FLAGS_STR "${CUDA_NVCC_FLAGS}")
-      set (CMAKE_CUDA_LINK_EXECUTABLE  "${CUDA_NVCC_EXECUTABLE}  <FLAGS> <LINK_FLAGS> <OBJECTS>  -o <TARGET> <LINK_LIBRARIES>" CACHE PATH "" )
+      set (CMAKE_CUDA_LINK_EXECUTABLE  "${CUDA_NVCC_EXECUTABLE}  <FLAGS> <LINK_FLAGS> <OBJECTS>  -o <TARGET> <LINK_LIBRARIES>" CACHE STRING "" )
       set_target_properties(${cuda_target} PROPERTIES LINKER_LANGUAGE "CUDA")
```

Comment:
hard to tell on quick review -- these stmts are already enclosed in BLT CHANGE comment blocks?

Reply:
this one isn't. Chris and I talked about at and we thought the change was minor enough to not warrant it...

Reply:
agreed